faithcomesbyhearing/dbp
# FCBH-1989 Searching My Library seems to take a long time

# Description
- Removed description of the notes when searching
- Improved search query on highlights bookmarks and notes by removing unnecessary `with` calls

## Issue Link
Original Story: [FCBH-1989](https://fullstacklabs.atlassian.net/browse/FCBH-1989) 

## How Do I QA This
- Run `http://dbp.test/api/search/library?v=4&key={KEY}&api_token={API_TOKEN}&query={QUERY}` and verify is working
